components/datadog/agent/docker/agentDockerInstaller.go

```diff
@@ -43,6 +47,7 @@ func NewAgentDockerInstaller(vm *vm.UnixVM, options ...dockerparams.Option) (*Ag
 		for key, value := range dockerAgentParams.Env {
 			env[key] = pulumi.String(value)
 		}
+		agentContainerName = "docker-on-vm-compose-tmp-agent-1" // TODO: Improve the naming and make it more robust
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should we rather assign a name in the docker compose using `container_name` ? --> https://docs.docker.com/compose/compose-file/compose-file-v3/#container_name

Reply:
Could be updated at https://github.com/DataDog/test-infra-definitions/blob/main/components/datadog/agent/docker.go#L14

Reply:
As it was trivial I did it in this PR.